nimble/controller/src/ble_ll_ctrl.c

```diff
@@ -1685,12 +1685,13 @@ ble_ll_ctrl_initiate_dle(struct ble_ll_conn_sm *connsm)
  * @param dptr
  * @param rspbuf
  * @param opcode
+ * @param initiate_dle
  *
  * @return int
  */
 static int
 ble_ll_ctrl_rx_feature_req(struct ble_ll_conn_sm *connsm, uint8_t *dptr,
-                           uint8_t *rspbuf, uint8_t opcode)
+                           uint8_t *rspbuf, uint8_t opcode, uint8_t *initiate_dle)
```

Comment:
I'd name this just as a generic flag to indicate that features were initialized by this LL op